narwhal/types/src/consensus.rs

```diff
@@ -63,6 +131,25 @@ impl ConsensusStore {
         write_batch.write()
     }
 
+    /// Persist a committed sub dag.
+    pub fn write_committed_sub_dag(
```

Comment:
Could you please add some instrumentation here? (maybe in `trace` level). I would be curious to keep track of the performance of this.

```
#[instrument(level = "trace", skip_all)]
```